components/mpas-ocean/src/Registry.xml

```diff
@@ -2068,6 +2204,18 @@
 		<var name="ssh" type="real" dimensions="nCells Time" units="m"
 			 description="sea surface height"
 		/>
+                <var name="verticalVelocity" type="real" dimensions="nVertLevelsP1 nCells Time" units="m s^{-1}"
+       		         description="prognostic vertical velocity, only used when config_enable_nonhydrostatic_mode = .true."
+    		/>
```

Comment:
There is already a `vertVelocityTop` that seems identical to this except that it is not prognostic.  
https://github.com/E3SM-Ocean-Discussion/E3SM/blob/6d69143ba38a5592c1be2097bd45b8e4a60a75a8/components/mpas-ocean/src/Registry.xml#L2794-L2797
It seems confusing to have both.  I wonder what we can do about this.

Reply:
@vanroekel, based on Xylar's comment, what do you think if we change the name from `verticalVelocity` to `vertVelocityNonhydro`?

Reply:
@xylar, I've talked with Luke and Darren, what do you think if we change the name from `verticalVelocity` to `vertVelocityNonhydro`?

Reply:
Sure, I think that works.